mypy.ini

```diff
@@ -18,6 +18,9 @@ disable_error_code =
 
 ## local
 
+# Use our custom stubs for distutils
+mypy_path = $MYPY_CONFIG_FILE_DIR/typings
```

Comment:
I used pyright's default of `typings` so that I didn't have to configure it there. But that folder name can be arbitrary.